src/test/suite/commands/launchMongoShell.test.ts

```diff
@@ -1,131 +1,128 @@
 import * as vscode from 'vscode';
-import * as sinon from 'sinon';
 import assert from 'assert';
 import { beforeEach, afterEach } from 'mocha';
+import sinon from 'sinon';
+import type { SinonSpy, SinonStub } from 'sinon';
 
 import launchMongoShell from '../../../commands/launchMongoShell';
 import { mdbTestExtension } from '../stubbableMdbExtension';
 
 suite('Commands Test Suite', () => {
-  void vscode.window.showInformationMessage('Starting tests...');
-
-  const mockConnectionController =
+  const testConnectionController =
     mdbTestExtension.testExtensionController._connectionController;
-  const sandbox = sinon.createSandbox();
 
-  let fakeShowErrorMessage: any;
-  let fakeGetMongoClientConnectionOptions: any;
-  let fakeIsCurrentlyConnected: any;
-  let createTerminalStub: any;
-  let fakeSendTerminalText: any;
+  let showErrorMessageStub: SinonStub;
+  let getMongoClientConnectionOptionsStub: SinonStub;
+  let isCurrentlyConnectedStub: SinonStub;
+  let createTerminalStub: SinonStub;
+  let sendTextStub: SinonSpy;
 
   beforeEach(() => {
-    sandbox.stub(vscode.window, 'showInformationMessage');
-
-    fakeShowErrorMessage = sandbox.stub(vscode.window, 'showErrorMessage');
-    fakeGetMongoClientConnectionOptions = sandbox.stub(
-      mockConnectionController,
+    sinon.stub(vscode.window, 'showInformationMessage');
```

Comment:
Why are we moving away from using `sandbox` at the top level here? If the goal here is for other tests outside this file to also be affected by this line, then this stubbing should be added to the relevant tests (or, if necessary, as a global `beforeEach`/`afterEach` hook pair)

Reply:
Even tho in this particular case, the clean up is handled in afterEach hook, I decided to move everything to sandbox as we discussed. I find tests now cleaner :) thx for sharing your point of view in DM!